Add sample data: Excel and SQL files

arryzheng opened the pull request.

NishArc reviewed (commented):
Thanks, Harry, for adding the sample data. Just noting that many of the columns (eg: UID15_CourseAdmissionsResKey, UID21_StudentLoansResKey, etc.) are populated with de-identified/encrypted values rather than real or realistic values. Since these won’t necessarily reflect the actual values we’d expect in production, this may limit the usefulness of the sample data for certain types of testing.

So, if you could please update the sample data to better reflect realistic dummy values, that would be helpful.

Thank you for your hard work!

arryzheng commented:
> Thanks, Harry, for adding the sample data. Just noting that many of the columns (eg: UID15_CourseAdmissionsResKey, UID21_StudentLoansResKey, etc.) are populated with de-identified/encrypted values rather than real or realistic values. Since these won’t necessarily reflect the actual values we’d expect in production, this may limit the usefulness of the sample data for certain types of testing.
> 
> So, if you could please update the sample data to better reflect realistic dummy values, that would be helpful.
> 
> Thank you for your hard work!

Thanks for the feedback, Maneesha. I’ve already updated the sample data on that, and replaced the encrypted values with more realistic dummy values to make the testing more useful. Please let me know if you spot anything else that could be improved.

arryzheng commented:
I have modified and tested the sample data, and it now perfectly fits the database schema with all table relationships successfully validated.
